docs/Changelog.md

```diff
@@ -14102,6 +14102,127 @@ This version of the operator has been available since version 12 of the default
 <dd>Constrain index tensor to int64</dd>
 </dl>
 
+### <a name="NllLoss-12"></a>**NllLoss-12**</a>
+
+  A NllLoss op computes (weighted) negative log likelihood loss.
+  Its "input" tensor has the shape of (N, C, d1, d2, ..., dk) where k >= 0. The "input" tensor contains log-probabilities for samples being in a class of [0, C).
+  The op's "target" input tensor has the shape of (N, d1, d2, ..., dk). It contains classifications (one of C classes) for N x d1 x d2 x ... x dk samples. 
+  The loss for a sample at n, d_1, d_2,...d_k being classified as class c = target[n][d_1][d_2]...[d_k] is computed as:
```

Comment:
```suggestion
  The loss for a sample at n, d_1, d_2,...d_k being classified as class c = target[n][d_1][d_2]...[d_k] is computed as:
```
`a sample at n, d_1, d_2,...d_k` is not a formal description. Do you mean `input[n, d_1, d_2, ..., d_k]`?